### PR TITLE
Unlock Arena War Trophies

### PR DESCRIPTION
**Pull request by ShinyWasabi**

### CODE REVIEW

File: WasabiWords.lua [hunk 1]
```diff
@@ -631,7 +747,7 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
         stats.set_int('MPX_LAST_ANIMAL', 108) -- Invade and Persuade II -- Kill 100 animals in a single playthrough
         stats.set_int('MPX_CH_ARC_CAB_CLAW_TROPHY', -1) -- Kitty Claw Trophy
         stats.set_int('MPX_CH_ARC_CAB_LOVE_TROPHY', -1) -- The Love Professor Trophy
-        stats.set_int('MPX_AWD_FACES_OF_DEATH', 50) -- Camhedz -- Faces Of Death
+        stats.set_int('MPX_AWD_FACES_OF_DEATH', 50) -- Camhedz -- Faces Of Death
```
Comment (gir489returns) [inline WasabiWords.lua:750]: Why?

File: WasabiWords.lua [hunk 2]
```diff
@@ -589,6 +589,122 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
...
+        stats.set_bool("MPX_AWD_JOCK", true)
+        stats.set_bool("MPX_AWD_WARRIOR", true)
+        stats.set_bool("MPX_AWD_T_BONE", true)
+        stats.set_bool("MPX_AWD_MAYHEM", true)
+        stats.set_bool("MPX_AWD_WRECKER", true)
+        stats.set_bool("MPX_AWD_CRASH_COURSE", true)
+        stats.set_bool("MPX_AWD_ARENA_LEGEND", true)
+        stats.set_bool("MPX_AWD_PEGASUS", true)
+        stats.set_packed_stat_bool(25010, true) -- Skip arena wall help.
+        stats.set_packed_stat_bool(25014, true) -- Skip arena wall tutorial.
```
Comment (gir489returns) [inline WasabiWords.lua:707]: This isn't unlocking anything that I can see in the scripts.

File: WasabiWords.lua [hunk 3]
```diff
@@ -746,4 +862,4 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
         globals.set_int(weekly_words_global + (1 + (0 * 6)) + 1, globals.get_int(weekly_words_global + (1 + (0 * 6)) + 2)) --Unlock Weekly Objective
         gui.show_message('WasabiWordsTM', 'Clichés Subverted')
     end)
-end)
+end)
```
Comment (gir489returns) [inline WasabiWords.lua:865]: Revert.

**Review by gir489returns — CHANGES_REQUESTED** [review]

File: WasabiWords.lua (hunk 2 above)
Comment (ShinyWasabi) [reply WasabiWords.lua:707]: Check `am_mp_arena_garage`, they are used to determine if you've seen the Arena War Wall help or tutorial before.

**Pull request merged**